modules/smtp/site.js

```diff
@@ -406,20 +444,45 @@ $(function() {
         $('.delete_draft').on("click", function() { smtp_delete_draft($(this).data('id')); });
         $('.smtp_save').on("click", function() { save_compose_state(false, true); });
         $('.smtp_send_archive').on("click", function() { send_archive(false, true); });
-        $('.compose_form').on('submit', function() {
-            var msg_uid = hm_msg_uid();
-            var detail = Hm_Utils.parse_folder_path(hm_list_path(), 'imap');
-            var class_name = 'imap_'+detail.server_id+'_'+msg_uid+'_'+detail.folder;
-            var key = 'imap_'+Hm_Utils.get_url_page_number()+'_'+hm_list_path();
-            var next_message = Hm_Message_List.prev_next_links(key, class_name)[1];
-            if (next_message) {
-                $('.compose_next_email_data').val(next_message);
+        $('.compose_form').on('submit', function(e) {
+            const compose_body_value = document.getElementById('compose_body').value;
+            const force_send = document.getElementById('force_send')?.value;
+
+            var reminder_value = $('.compose_form').data('reminder');
+
+            if (reminder_value === 1) {
+                const uploaded_files = $("input[name='uploaded_files[]']").map(function () { return $(this).val(); }).get();
+                const translated_keywords = 'attachment,file,attach,attached,attaching,enclosed,CV,cover letter' + //en
```

Comment:
It would be better to make this list translatable and put into translation arrays for each language.

Reply:
Copy that @kroky, Refactor Done.